test/unit/carriers/usps_test.rb

```diff
@@ -386,6 +386,13 @@ def test_strip_9_digit_zip_codes
     assert request =~ /\>12345\</
   end
 
+  def test_strip_9_digit_zip_codes_world_rates
+    request = URI.decode(@carrier.send(:build_world_rate_request, location_fixtures[:beverly_hills_9_zip],
+                                        package_fixtures[:book], location_fixtures[:auckland], {}))
+    assert !(request =~ /\>90210-1234\</)
```

Comment:
I recommend using http://apidock.com/ruby/MiniTest/Assertions/assert_matchhttp://apidock.com/ruby/MiniTest/Assertions/assert_match, not the generic assert.